src/main/java/de/hbt/pwr/profile/service/SkillRecommendationService.java

```diff
@@ -54,6 +61,7 @@
                 .flatMap(Collection::stream)
                 .filter(s -> (!project.getSkills().contains(s)))
                 .distinct()
+                .sorted(new SkillNameComparator())
```

Comment:
```suggestion
                .sorted(Comparator.comparing(Skill::getName))
```
Spart die die anonyme Klasse und ist genau dafür gemacht